### ISSUE TITLE
Access modes and presentational content

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#38]
We don't explicitly state whether to set access modes for content that is purely presentational.

If a book has a few decorative images, for example, does it still have a visual access mode? Does it have both a visual and textual sufficient access mode and a textual one, or does it only have a textual access mode?

I've always assumed that you wouldn't set an access mode for presentational content, but I'm going solely by the word "information" in the definition.

Would help to be clearer on this.

**Comment by mattgarrish** [comment]
For a thought experiment... pretty much all commercial books have covers, but knowing what's on the cover isn't necessary to read the book itself. So do books whose only imagery is their cover have a visual access mode? If the cover doesn't have alt text and/or isn't described, does it make sense to say that the book doesn't have a textual-only sufficient access mode?

Or put another way, how do we deal with imagery like covers, maybe plates, etc. that aren't fully presentational but also aren't directly contributing to the "information" of the work?

**Comment by clapierre** [comment]
What we have been doing in GCA is the following, only "Content Images" within the EPUB would constitute an accessMode of "visual" and similarly accessModeSufficient "visual/textual" when described. We do not include the cover image as then that would be every EPUB making this metadata useless for EPUBs.  We also do not consider Logo's / Copyright images and a stretch the Author's image if present, however we do add that these exist and have alt text descriptions in the accessibilitySummary.

**Comment by madeleinerothberg** [comment]
I agree -- presentational or decorative images that don't add any "information" to the book should not be reported.
